src/lib.rs

```diff
@@ -96,15 +98,22 @@ pub mod prelude {
 /// ```
 pub fn spawn<T: Send + 'static>(future: impl Future<Output = T> + Send + 'static) -> Task<T> {
     static GLOBAL: Lazy<Executor> = Lazy::new(|| {
-        thread::Builder::new()
-            .name("smol".to_string())
-            .spawn(|| {
-                loop {
+        let num_threads = {
+            // Parse SMOL_THREADS or use the number of CPU cores on the system.
```

Comment:
"use the number of CPU cores on the system" is not correct ;)

Reply:
Heh, too much copy/paste :)